src/mesh/generated/mesh.pb.h

```diff
@@ -73,6 +73,8 @@ typedef enum _HardwareModel {
     HardwareModel_M5STACK = 44, 
     /* B&Q Consulting Station Edition G1: https://uniteng.com/wiki/doku.php?id=meshtastic:station */
     HardwareModel_STATION_G1 = 45, 
+    /* TODO: REPLACE */
+    HardwareModel_LILYGO_TBEAM_S3_CORE = 46,
```

Comment:
Since it's safe to assume another variant of the T-Beam will be a new core product, i suggest moving that from 46 to index 12. Also this file is auto-generated. This change needs to be made through meshtastic-protobufs repository.

Reply:
Thanks ! I have seen the changes in meshtastic-protobufs